lib/app/components/organisms/multi_option_select_button.dart

```diff
@@ -63,5 +61,25 @@ class MultiOptionSelectButton extends StatelessWidget{
       ],
     );
   }
+
+  Widget _buildItems(List<Widget> items, Widget? emptyChild){
+    if(items.isNotEmpty){
+      return Wrap(
+        runSpacing: 4.0,
+        spacing: 4.0,
+        children: items,
+      );
+    }
+    else if(emptyChild != null){
+      return emptyChild;
+    }
+    else{
+      return Wrap(
+        runSpacing: 4.0,
+        spacing: 4.0,
+        children: const <Widget>[],
+      );
+    }
+  }
```

Comment:
```suggestion
    if(emptyChild != null){
      return emptyChild;
    }
    return Wrap(
        runSpacing: 4.0,
        spacing: 4.0,
        children: const <Widget>[],
      );
    }
```
早期退出するならelse抜きにしましょう